mos-platform/nes-action53/_prg-rom-banked.ld

```diff
@@ -15,10 +15,10 @@ MEMORY {
   prg_rom_0     : ORIGIN = __prg_rom_0_lma,     LENGTH = __prg_rom_size >= 32  ? 0x4000 : 0
   prg_rom_1     : ORIGIN = __prg_rom_1_lma,     LENGTH = __prg_rom_size >= 64  ? 0x4000 : 0
   prg_rom_2     : ORIGIN = __prg_rom_2_lma,     LENGTH = __prg_rom_size >= 64  ? 0x4000 : 0
-  prg_rom_fixed : ORIGIN = __prg_rom_fixed_lma, LENGTH = 0x4000 - 0x30
+  prg_rom_fixed : ORIGIN = __prg_rom_fixed_lma, LENGTH = 0x4000 - 0x32
 
   /* Reset stub for the fixed bank. */
-  reset_stub : ORIGIN = 0x10000 - 0x30, LENGTH = 0x10
```

Comment:
No Action Required for this change, but does reset_stub actually have to be the very last thing in the address space, or can it just be assigned to prg_rom_fixed? The latter would save needing to adjust this if the stub changes sizes.

Reply:
It likely should, good point. This was a hold over from when i was first starting to debug the build and was testing out linker sections.